blocks/footer/footer.js

```diff
@@ -0,0 +1,176 @@
+import {
+  fetchVariables,
+} from '../../scripts/scripts.js';
+
+async function markupToFooter(url) {
+  const resp = await fetch(`${url}.plain.html`);
+  const html = await resp.text();
+  const placeholder = document.createElement('div');
+  placeholder.innerHTML = html;
+  const data = {};
+
+  const regionEl = placeholder.querySelector('.region-selector');
```

Comment:
i am actually thinking that those should be somewhere in the footer content. especially for `business.adobe.com/blog/` those will be empty initially.

Reply:
Yup. Same with the regions below. All of this really should be lazily loaded, or better: on demand.

I talked to Porter about this issue and we both agreed to flash a modal when a person clicks on the region selector explaining content isn't available yet.

Reply:
@auniverseaway Any ETA on when the 'content not available' text will be available (and localized)?

Reply:
@fkakatie I'd say ship as is. I'll address after you merge this in.